lib/openstax/aws/stack.rb

```diff
@@ -319,6 +328,10 @@ def deleting?
       "DELETE_IN_PROGRESS" == status
     end
 
+    def exists?
```

Comment:
a (simple) unit test for this?

Reply:
Yeah sorry didn't mean for you to review this until I had written the test.